lib/client/boluscalc.js

```diff
@@ -83,11 +83,13 @@ function init(client, $, plugins) {
   }
   
   boluscalc.updateVisualisations = function updateVisualisations (sbx) {
-    boluscalc.lastsbx = sbx;
+    // update BG in GUI
+    setBG(sbx.lastSGVEntry(), mergeDateAndTime().toDate());
+    
+    if (client.browserUtils.getLastOpenedDrawer != '#boluscalcDrawer') {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1744870305/issues/source?bid=2452379&fileBranchId=3059293#l89)